active_storage_base64.gemspec

```diff
@@ -18,6 +18,8 @@ Gem::Specification.new do |s|
 
   # Dependencies
   s.add_dependency 'rails', '~> 5.2.0', '<= 5.2.2'
+  s.add_dependency 'mime-types', '~> 3.0'
```

Comment:
oh I just see.. are you using these 2 new dependencies?

Reply:
Oh, yeah, they were for using the MimeTypes approach from Carfrierwave, but we were gonna let ActiveStorage take care of the type inference, so I can remove these two! Good catch